rails6-example/Gemfile

```diff
@@ -3,6 +3,9 @@ git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
 gem 'database_consistency', path: '../', group: :development, require: false
 
+# https://github.com/rails/rails/pull/54264
```

Comment:
many applications including iOS applications broke with the recent concurrent-ruby release